### DISCUSSION TITLE
Support for Wireguard

### DISCUSSION THREAD

**Discussion by crisconru** [elementary/triage#345] [Ideas]
Wireguard is maybe the faster and more secure VPN and easier to use technology at this moment (I think it's even included in the kernel of Linux). It is impossible to setup in the System Settings -> Network.

There are a pull request in the Swtichboard-plug-network -> [link](https://github.com/elementary/switchboard-plug-network/pull/338)

I have used Wireguard for 3 years and my main (and only) OS is Elementary. I always need to go to the terminal to open the vpn and close it. It would be nice to have a user friendly integration

**Comment by danirabbit** [comment]
As far as I can tell, the Wireguard branches have been merged so you should expect to see Wireguard support in OS 8! Thanks for your suggestions :)
